uint64 packet_timestamp
uint8 stage
int32 stage_time_left
uint8 command
uint32 command_counter
uint64 command_timestamp
team_info yellow
team_info blue
